packages/quantic/force-app/main/default/lwc/quanticAriaLive/__tests__/quanticAriaLive.test.js

```diff
@@ -0,0 +1,106 @@
+import {cleanup, buildCreateTestComponent, flushPromises} from 'c/testUtils';
+import QuanticAriaLive from 'c/quanticAriaLive';
+
+const createTestComponent = buildCreateTestComponent(
+  QuanticAriaLive,
+  'c-quantic-aria-live'
+);
+
+describe('c-quantic-aria-live', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  describe('registerRegion', () => {
+    it('should register a new region with default values', async () => {
+      const element = createTestComponent();
+      element.registerRegion('testRegion');
+      await flushPromises();
+
+      const region = element.shadowRoot.querySelector(
+        '[data-key="testRegion"]'
+      );
+      expect(region).not.toBeNull();
+      expect(region.getAttribute('aria-live')).toBe('polite');
+      expect(region.textContent).toBe('');
+    });
+
+    it('should register a new region with assertive set to true', async () => {
+      const element = createTestComponent();
+      element.registerRegion('testRegion', true);
+      await flushPromises();
+
+      const region = element.shadowRoot.querySelector(
+        '[data-key="testRegion"]'
+      );
+      expect(region).not.toBeNull();
+      expect(region.getAttribute('aria-live')).toBe('assertive');
+      expect(region.textContent).toBe('');
+    });
+
+    it('should not overwrite an existing region', async () => {
+      const element = createTestComponent();
+      element.registerRegion('testRegion');
```

Comment:
for this test do you want also to check if a new value for the region is set?

Reply:
This test is about registering the regions, no value is set yet,